k8s/k8s.go

```diff
@@ -35,12 +37,12 @@ func (s *k8sSecrets) GetSecret(
 
 	secret, err := core.Instance().GetSecret(secretName, namespace)
 	if err != nil {
+		if strings.Contains(err.Error(), "not found") {
```

Comment:
How about using `kerrors.IsNotFound(err)` Where `kerrors` comes from `kerrors "k8s.io/apimachinery/pkg/api/errors"`. Perhaps `core` has a similar function?

Reply:
Even better 😊, pushed `kerrors.IsNotFound(err)` and unit test.

Reply:
Are github actions disabled on this repository?

Reply:
Not really but a maintainer must run the job manually first since you are a first-time contributor

Reply:
I don't see this `kerrors.IsNotFound(err)` change in the latest version of PR. Btw I have triggered the github actions on this PR.